examples/public/images/icons/about.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 45 40" enable-background="new 0 0 45 40" xml:space="preserve">
+<g id="guides">
+</g>
+<g id="Layer_2">
+	<g>
+		<path d="M22.5,10.1c5.2,0,9.4,4.2,9.4,9.4s-4.2,9.4-9.4,9.4s-9.4-4.2-9.4-9.4S17.3,10.1,22.5,10.1 M22.5,8.6
+			c-6,0-10.9,4.9-10.9,10.9s4.9,10.9,10.9,10.9s10.9-4.9,10.9-10.9S28.5,8.6,22.5,8.6L22.5,8.6z"/>
+	</g>
+	<g>
+		<g>
+			<path d="M24.1,22.8c0,0,0.1,0.1,0,0.2c-0.1,0.1-1,1.6-3,1.6c-0.5,0-0.6-0.2-0.6-0.5c0-0.3,0.3-1,0.9-2.9l0.4-1.2

```

Comment:
These are Adobe Illustrator exports. I really don't want to do "code cleanup" on an image file.